streaming.md

```diff
@@ -138,47 +210,53 @@ FlinkPravegaWriter<MyClass> pravegaSink = FlinkPravegaWriter.<MyClass>builder()
-|`withTxnLeaseRenewalPeriod`|The transaction lease renewal period that supports the exactly-once writer mode.|
-|`withSerializationSchema`|The serialization schema which describes how to turn events into byte messages.|
-|`withEventRouter`|The router function which determines the routing key for a given event.|
-|`enableMetrics`|true or false to enable/disable reporting Pravega reader group metrics. By default, the metrics option is enabled.|
+## Serialization
+Please, see the [serialization](serialization) page for more information on how to use the serializer.
 
 ### Parallelism
-The data sink supports parallelization.  Use the `setParallelism` method to configure the number of parallel instances to execute.
+FlinkPravegaWriter supports parallelization. For more information please revist the concept [Parallelism](#parallelism) discussed in the above section.
 
 ### Event Routing
-Every event written to a Pravega stream has an associated routing key.  The routing key is the basis for event ordering.  See the [Pravega documentation](http://pravega.io/docs/pravega-concepts/#events) for details.
+Every event written to a Pravega stream has an associated Routing Key.  The Routing Key is the basis for event ordering.  See the [Pravega documentation](http://pravega.io/docs/pravega-concepts/#events) for details.
 
-To establish the routing key for each event, provide an implementation of `io.pravega.connectors.flink.PravegaEventRouter` when constructing the writer.
+When constructing the FlinkPravegaWriter, to establish the Routing Key for each event, provide the below mentioned implementation:
+```
+io.pravega.connectors.flink.PravegaEventRouter` 
+```
 
 ### Event Time Ordering
-For programs that use Flink's event time semantics, the connector library supports writing events in event time order.   In combination with a routing key, this establishes a well-understood ordering for each key in the output stream.
 
-Use the `FlinkPravegaUtils::writeToPravegaInEventTimeOrder` method to write a given `DataStream` to a Pravega stream such that events are automatically ordered by event time (on a per-key basis). Refer [here](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/test/java/io/pravega/connectors/flink/FlinkPravegaWriterITCase.java#L93) for the sample code.
+For programs that use Flink's event time semantics, the connector library supports writing events in event time order. In combination with a Routing Key, this establishes a well-understood ordering for each key in the output stream.
+
+Use the following method to write a given `DataStream` to a Pravega stream such that events are automatically ordered by event time (on a per-key basis). ample code can be found [here](https://github.com/pravega/flink-connectors/blob/7971206038b51b3cf0e317e194c552c4646e5c20/src/test/java/io/pravega/connectors/flink/FlinkPravegaWriterITCase.java#L93) for the sample code.

```

Comment:
type -> ample code and also remove "for the sample code"

Reply:
Edited,please check